attestation-service/README.md

```diff
@@ -134,16 +159,19 @@ Supported Verifier Drivers:
 
 The AS supports modular policy engine, which can be specified through the AS configuration. The currently supported policy engines are:
```

Comment:
Should there be a mention that the policy is universal and no support for multiple policies?

Reply:
There is support for multiple policies. I am going to change this sentence around though. I don't think the modularity of the policy engine (i.e. the possibility to implement something other than OPA) is very significant.